services/st.cmd

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+export PATH=/opt/conda/bin/:$PATH
+export USER=softioc
+export HOME=/epics/iocs/notebook
+export SHELL=bash
+
+export EPICS_CA_AUTO_ADDR_LIST="no"
+export EPICS_CA_ADDR_LIST="10.66.131.255"
```

Comment:
suggest replacing these with 1 of (from most general to specific):

```
. /etc/profile
```
```
# relevant snippet from /etc/profile
for i in /etc/profile.d/*.sh /etc/profile.d/sh.local ; do
    if [ -r "$i" ]; then
        if [ "${-#*i}" != "$-" ]; then
            . "$i"
        else
            . "$i" >/dev/null
        fi
    fi
done
```
```
. /etc/profile.d/epics.sh
```
The 1st would seem closest to what you'd get starting a notebook from a terminal.

Reply:
```suggestion
. /etc/profile
```

Reply:
or

Reply:
```suggestion
. /etc/profile.d/epics.sh
```

Reply:
I like the last option. Thanks, @cowanml!